### PR TITLE
Cards #57 Dashboard, #61 Global View and #9 Excel

### PR DESCRIPTION
**Pull request by carehs12**
Changes Made
#57: 
- Dashboard components to show states, attachments, updated quotations, comments and quotations to be expired soon
- Added Dashboard to navbar and removed unused anchors
#61
- Added Anchors to navigate through the views in the quotations module
- Added requested changes to the global view (logo, footer, etc.)
#9 Generate Excel
- Generated final excel and uploaded templates

### CODE REVIEW

**Comment by carehs12** [comment]
Changes made (October 28):
- Changed some translations
- Removed 'credits' field
- Fixed some issues in the 'products' form of the quotation
-  Hide materials table when there are no materials in the global view
- The user can no longer edit services that have been deleted
- Fixed issues when a deleted product was on the quotation

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # Ignore uploaded files in development.
 /storage/*
 !/storage/.keep
+!/storage/template-comparative.xlsx
+!/storage/template-simple.xlsx
+!/storage/plantilla_precios.xlsx
```
Comment (ldonis) [inline .gitignore]: - Dentro de storage crear un folder llamado "templates" colocar adentro todas las plantillas
- Utilizar unicamente nombres en ingles

File: app/controllers/quotations_controller.rb [hunk 2]
```diff
@@ -102,13 +103,22 @@ def api_activate
     end
   end
 
+  def generate_excel
+    workbook = CreateExcelQuotation.new(id: params[:id]).create
+    #p file_path
+    code = (@quotation.id+100)
+    client = @quotation.client.name
+    filename = Date.today.to_s.gsub(/-/,'')+"-"+code.to_s+"-"+client+".xlsx"
```
Comment (ldonis) [inline app/controllers/quotations_controller.rb]: Utilizar strftime para parsear fechas
https://apidock.com/ruby/DateTime/strftime

File: app/controllers/dashboards_controller.rb [hunk 3]
```diff
@@ -1,9 +1,58 @@
...
+      })  
+    }
+    response_with_success(last_comment)
+  end
+
+  def api_recent_attachments
+    last_attachments = Attachment.order(created_at: "desc").limit(5).map {|attachment|
+    attachment.attributes.merge({
+      user: attachment.user.user_detail.name,
+      quotation: attachment.quotation.id+100,
```
Comment (ldonis) [inline app/controllers/dashboards_controller.rb]: Quitar el incremental

File: app/controllers/dashboards_controller.rb [hunk 4]
```diff
@@ -1,9 +1,58 @@
...
+    }
+    response_with_success(last_attachments)
+  end
+
+  def api_recent_quotations
+    last_quotations_updated = Quotation.order(updated_at: "desc").limit(5).map {|attachment| 
+      attachment.attributes.merge({
+        user: attachment.user.user_detail.name,
+        client: attachment.client.name,
+        quotation: attachment.id+100,
```
Comment (ldonis) [inline app/controllers/dashboards_controller.rb]: Quitar el "+100"

File: app/controllers/products_controller.rb [hunk 5]
```diff
@@ -59,6 +59,12 @@ def upload
     redirect_to products_path, notice: t('.upload')
   end
 
+  def download_price
+    workbook = CreateExcelProducts.new().create
+    filename = Date.today.to_s+".xlsx"
```
Comment (ldonis) [inline app/controllers/products_controller.rb]: Agregar nombre mas descriptivo al excel. Ej. "Precio de productos FECHA.xlsx"

**Pull request merged**